### PR TITLE
add automated setup badge and add .gitpod.yml patch

### PR DESCRIPTION
**Pull request by JesterOrNot**
Added new Gitpod automated setup badge and changed gitpod setup to build nu from source code rather than crates.io

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 image:
   file: .gitpod.Dockerfile
 tasks:
-  - init: cargo install nu --features=stable
+  - init: cargo install --path . --force
```
Comment (jonathandturner) [inline .gitpod.yml]: Should this be --features=stable also?

Comment (JesterOrNot) [reply .gitpod.yml]: Good catch!

**Comment by JesterOrNot** [comment]
Take another look?

**Comment by jonathandturner** [comment]
Looks good

**Pull request merged**